velox/connectors/tpch/tests/TpchConnectorTest.cpp

```diff
@@ -41,6 +41,7 @@ class TpchConnectorTest : public exec::test::OperatorTestBase {
             connector::tpch::TpchConnectorFactory::kTpchConnectorName)
             ->newConnector(kTpchConnectorId, nullptr);
     connector::registerConnector(tpchConnector);
+    aggregate::prestosql::registerAllAggFunctions();
```

Comment:
Let's move this call next to registerAllScalarFunctions in OperatorTestBase::SetUpTestCase.

naming: registerAllAggFunctions -> registerAllAggregateFunctions

```
void OperatorTestBase::SetUpTestCase() {
  functions::prestosql::registerAllScalarFunctions();
  TestValue::enable();
}
```